eos-tech-support/eos-write-live-image

```diff
@@ -484,6 +491,15 @@ if [ "$EXPAND" ]; then
     fi
 fi
 
+if [ "$PERSISTENT" ]; then
+    FREE_SPACE_BYTES=$(df -P -B1 "$DIR_IMAGES_ENDLESS" | awk 'NR==2 {print $4}')
```

Comment:
What happens if this is not a multiple of 512? (I think that's the sector size device-mapper requires). Is it guaranteed to be a multiple of 512 thanks to some property of NTFS or exFAT?

Reply:
It appears that on exFAT the free space is always* a multiple of some larger power of 2 than 512.

* I did a couple of random tests.

Being a FAT based filesystem this is probably always going to be true, but I'm just waving my hands.  Would you prefer explicitly enforcing a 512 byte alignment just in case?

Reply:
A comment to this effect is probably OK! Or you could divide by 1024 and specify the unit as K when calling `truncate`. Probably overengineering though.

Reply:
I've pushed changes to do the division thing... (I originally wanted to use df -B1024, but it rounds up)